src/Components/Map/Map.tsx

```diff
@@ -10,18 +11,96 @@ interface MapWithGoogleProps {
 }
 const libraries = ['places'];
 
+const dataLayer = (window as any).dataLayer || [];
```

Comment:
I may be missing it, but I don't see the Google Analytics code that stores the lat/long.